tutorial_sarscov2.md

```diff
@@ -0,0 +1,259 @@
+> - A second level for distinction of datasets belonging to the same sample (e.g.: sample dates).
+> - Inside that directory, the sub-directory `raw_data` holds the sequencing output in FASTQ format (optionally compressed with GZip).
+> - When in split files, paired-ends reads need to have `_R1` and `_R2` suffixes in their name.
+
+*[FASTQ]: [represents DNA sequencer reads along with quality scores](https://en.wikipedia.org/wiki/FASTQ_format)
+
+## Preparing a small dataset
+
+You can run the first test on your workstation or a good laptop.
+
+First, you need to prepare the data:
+
+ - For that test, you need to download the following runs from SRA:
+   [SRR10903401](https://trace.ncbi.nlm.nih.gov/Traces/sra/?run=SRR10903401) and
+   [SRR10903402](https://trace.ncbi.nlm.nih.gov/Traces/sra/?run=SRR10903402)
+   
+   (hint: click on the _data_ tab to find the download links).
+ - Then, organize the downloaded files to have the following structure
+   (you need to rename the files so that they have `_R1` and `_R2` suffixes):
+
+
+*[SRA]: [sequence read archive](https://trace.ncbi.nlm.nih.gov/Traces/sra/)
+
+```
+samples
+├── SRR10903401
+│   └── 20200102
+│       └── raw_data
+│           ├── wuhan2_R1.fastq
+│           └── wuhan2_R2.fastq
+└── SRR10903402
+    └── 20200102
+        └── raw_data
+            ├── wuhan1_R1.fastq
+            └── wuhan1_R2.fastq
+```
+
+## Install V-pipe
+
+V-pipe uses the [Bioconda](https://bioconda.github.io/)[^bioconda] bioinformatics software repository for all its pipeline components.
+The pipeline itself is written using [snakemake](https://snakemake.readthedocs.io/)[^Snakemake].
+
+[^bioconda]: Grüning, Björn, Ryan Dale, Andreas Sjödin, Brad A. Chapman, Jillian Rowe, Christopher H. Tomkins-Tinch, Renan Valieris, the Bioconda Team, and Johannes Köster. 2018. “Bioconda: Sustainable and Comprehensive Software Distribution for the Life Sciences”. Nature Methods, 2018 doi:[10.1038/s41592-018-0046-7](https://doi.org/10.1038/s41592-018-0046-7).
+
+[^Snakemake]: Johannes Köster and Sven Rahmann. [Snakemake – a scalable bioinformatics workflow engine](https://academic.oup.com/bioinformatics/article/28/19/2520/290322). *Bioinformatics*, 28(19):2520–2522, 2012. doi:[10.1093/bioinformatics/bts480](https://doi.org/10.1093/bioinformatics/bts480)
+
+> **For advanced users:** If your are fluent with these tools, you can:
+> - directly download and install
+>   [bioconda](https://bioconda.github.io/user/install.html) and
+>   [snakemake](https://snakemake.readthedocs.io/en/stable/getting_started/installation.html#installation-via-conda),
+> - clone the [sars-cov2 branch of V-pipe](https://github.com/cbg-ethz/V-pipe/tree/sars-cov2)
+> - and start using V-pipe with them, using the `--use-conda` to
+>   [automatically download and install](https://snakemake.readthedocs.io/en/stable/snakefiles/deployment.html#integrated-package-management)
+>   any [further pipeline dependencies]({{ "/pipeline/" | relative_url }}).
```

Comment:
+ "please refer to the documentation for additional instructions (https://github.com/cbg-ethz/V-pipe/wiki/getting-started#running-v-pipe)"